setup.py

```diff
@@ -1,48 +1,48 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import os
 import codecs
+import os
+
 from setuptools import setup
 
 
 def read(fname):
     file_path = os.path.join(os.path.dirname(__file__), fname)
-    return codecs.open(file_path, encoding='utf-8').read()
+    return codecs.open(file_path, encoding="utf-8").read()
 
 
 setup(
-    name='pytest-pyvista',
-    version='0.1.0',
-    author='The PyVista Developers',
-    author_email='alex.fernandezluces@ansys.com',
-    maintainer='The PyVista Developers',
-    maintainer_email='info@pyvista.org',
-    license='MIT',
-    url='https://github.com/pyvista/pytest-pyvista',
-    description='Plugin to test PyVista plot outputs',
-    long_description=read('README.rst'),
-    py_modules=['pytest_pyvista'],
-    python_requires='>=3.5',
-    install_requires=['pytest>=3.5.0'],
+    name="pytest-pyvista",
+    version="0.1.0",
+    author="The PyVista Developers",
+    author_email="info@pyvista.org",
+    maintainer="The PyVista Developers",
+    maintainer_email="info@pyvista.org",
+    license="MIT",
+    url="https://github.com/pyvista/pytest-pyvista",
+    description="Plugin to test PyVista plot outputs",
+    long_description=read("README.rst"),
+    py_modules=["pytest_pyvista"],
+    python_requires=">=3.7",
+    install_requires=["pytest>=3.5.0"],
```

Comment:
Should PyVista be listed as a requirement as well?

Reply:
Yes, since it is used in the tests. It is added now.